src/compiler/utilities.ts

```diff
@@ -11644,6 +11644,14 @@ export function unwrapParenthesizedExpression(o: Expression) {
     return o;
 }
 
+/** @internal */
+export function unwrapParenthesizedType(o: TypeNode) {

```

Comment:
I'm surprised we don't already have this helper...

Reply:
I was surprised too. But I could not find it if it exists. There is one that walks up.

Reply:
Ah, it's `skipTypeParentheses`.

Reply:
Oh. Didn't think to look for that. I'll fix it tomorrow.